airbyte-integrations/connectors/destination-snowflake/metadata.yaml

```diff
@@ -28,6 +28,11 @@ data:
   supportsDbt: true
   tags:
     - language:java
+  releases:
+    breakingChanges:
+      2.0.0:
+        message: "Remove GCS/S3 loading method support."
+        upgradeDeadline: "2023-08-31"
```

Comment:
@evantahler does this only apply to Cloud? 

This date is arbitrary today as no one is using these loading methods.

Reply:
It will apply to OSS too (in that they will see the alert), but on OSS we won't pause syncs once the date passes.  We will on cloud until you update.